### DISCUSSION TITLE
Blob Limits

### DISCUSSION THREAD

**Discussion by hmessing** [vercel/next.js#63145] [Help]
### Summary

For blobs, what is the maximum number of simultaneous connections?

### Additional information

```js
I'm writing an app that generates and uses a lot of thumbnails from pictures stored as blobs. I occasionally get a message indicating "too many simultaneous connections." What are the limits?
```

**Comment by DevDaniloFerrari** [comment]
Certainly! If you can share the code snippet where you connect to the blob and generate the thumbnail, I'd be happy to take a look and help you with it.

**Comment by hmessing** [comment]
I don't think I need help with the code. Solving the problem by putting a
10ms delay on each request, which works for my app, is easy enough.

I was just wondering what the hard limits are so I can achieve maximum
performance without exceeding them.

If that is not something you want to share, that's ok...your service is
great...

Thanks.

Howard
